examples/echo-test/echo-test

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
```

Comment:
Why not a `matrix-echo` bin under the tool directory and add it in the installable scripts list (https://github.com/psaavedra/matrix-bot/blob/master/setup.py#L51) .